source/fundamentals/indexes.txt

```diff
@@ -371,3 +384,53 @@ The following example creates a unique, descending index on the ``theaterId`` fi
 
 Refer to the :manual:`Unique Indexes page </core/index-unique>` in the MongoDB server manual for more information.
 .. driver-content-end
+
+Remove an Index
+---------------
+
+Pass the indexed ``name`` field to the ``dropIndex()`` method to remove
+an index from a collection. 
+
+Use the ``listIndexes()`` method to see the indexed ``name`` fields in
+your collection. 
+
+The following snippet retrieves all the indexes in your collection:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin listIndex
+   :end-before: end listIndex
+
+Your output should look something like the following:
+
+.. code-block:: json
+
+   { "v": 2, "key": {"_id": 1}, "name": "_id_" }
+   { "v": 2, "key": {"_fts": "text", "_ftsx": 1}, "name": "title_text", "weights": {"title": 1}, 
+   "default_language": "english", "language_override": "language", "textIndexVersion": 3 }
+
+The output preceding tells us there is a text index with the name "title_text".
+
+The following removes the ``title_text`` index in the collection:
+
+.. code-block:: java
+
+   collection.dropIndex("title_text");
+
+.. note:: 
+
+   You cannot remove a single index from a compound index. You must remove
+   all the indexes in your compound index.
+
+Removing All Indexes
+~~~~~~~~~~~~~~~~~~~~
+
+Starting with MongoDB 4.2, you can drop all non-``_id`` indexes in a
+collection using the ``dropIndexes()`` method on your collection. For
+MongoDB versions prior, use ``dropIndex("*")`` on your collection.
```

Comment:
I think this could be a better way to explain it since you're mixing code examples (with parameters) and method names without letting the user know the difference between them.

```suggestion
collection by calling the ``dropIndexes()`` method on your collection. For
MongoDB versions prior, pass the "*" as a parameter to your call to ``dropIndex()`` on your collection.
```

It may be better to just use code blocks for both so that it's consistent with your prior dropIndex example.